app/helpers/about_helper.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+module AboutHelper
+  DEVELOPER = [{ name: 'ihor_moskvyn',
```

Comment:
if it's singular `DEVELOPER`, the array shouldn't be used
otherwise `DEVELOPERS`

Reply:
done